src/node.cc

```diff
@@ -2565,7 +2563,8 @@ static void DebugProcess(const FunctionCallbackInfo<Value>& args) {
     goto out;
   }
 
-  pid = (DWORD) args[0]->IntegerValue();
+  CHECK(args[0]->IsNumber());
+  DWORD pid = (DWORD) args[0].As<Integer>()->Value();
```

Comment:
If you happen to be able to test it on Windows without much extra work – I think the cast here is unnecessary.